ClassySchedule/src/main/java/scheduler/AddCourseToScheduleController.java

```diff
@@ -146,6 +171,29 @@ public void initialize(URL url, ResourceBundle resourceBundle) {
             e.printStackTrace();
         }
 
+        // Initialize the invalid error and hide them.
+        invalidClassName.setTitle("Invalid Class Name Error");
```

Comment:
Do we need to initialize and then hide them all here? Or can we initialize them when we actually create the error?